src/Sylius/Bundle/AdminBundle/Resources/views/Grid/Field/state.html.twig

```diff
@@ -1,7 +1,7 @@
 {% set value = 'sylius.ui.' ~ data %}
 
 {% if options.vars.labels is defined %}
-    {% include [(options.vars.labels ~ ':' ~ data ~ '.html.twig'), '@SyliusUi/Label/_default.html.twig'] with {'value': value} %}
+    {% include [(options.vars.labels ~ '/' ~ data ~ '.html.twig'), '@SyliusUi/Label/_default.html.twig'] with {'value': value} %}
```

Comment:
We can use only this line instead of the whole `if` structure (don't know how it will affect performance though).